ide/src/main.rs

```diff
@@ -7,5 +9,11 @@ mod window;
 fn main() {
     env_logger::init();
 
-    Editor::new().unwrap().run();
+    let (event_loop, mut editor) = Editor::new().unwrap();
```

Comment:
Do we really want to unwrap here?

Reply:
There isn't much else to do except log the error, but the unwrap should to that.